src/main/java/lebron/MarkCommand.java

```diff
@@ -0,0 +1,20 @@
+package lebron;
+
+public class MarkCommand extends Command {
+
+    private int index;
+
+    public MarkCommand(int index) {
+        this.index = index;
+    }
+
+    @Override
+    public String execute(TaskList taskList, Ui ui, Storage storage) throws LeBronException {
+        Task task = taskList.getTask(index);
+        taskList.markTask(index);
+        String msg = ui.showTaskMarked(task);
```

Comment:
You could consider using the full name for `msg` to avoid "texting style" variable names, perchance?